Website/src/items/content/EdgeWebsite/home/speakers/Page Components/All speakers Section.yml

```diff
@@ -0,0 +1,40 @@
+﻿---
+ID: "54128e66-c795-4816-9a7a-59ceb06dea90"
+Parent: "81347f0c-a6f8-4a09-8d0a-0600485d5894"
+Template: "1f476f1c-822a-5a88-8f7e-2431b77d1b70"
+Path: /sitecore/content/EdgeWebsite/home/speakers/Page Components/All speakers Section
+Languages:
+- Language: en
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "001dd393-96c5-490b-924a-b0f25cd9efd8"
+      Hint: __Lock
+      Value: <r />
+    - ID: "05607296-e17f-567f-898f-46bb9665d619"
+      Hint: title
+      Value: Other event Speakers
```

Comment:
Either all words should have an uppercase first letter or Speakers should have a lowercase first letter. Please align with other section titles on the website.